### DISCUSSION TITLE
Option to map mod key alone.

### DISCUSSION THREAD

**Discussion by vunhatchuong** [qtile/qtile#3942] [Ideas]
So Pop! OS has thing where you can hit mod4 alone without any other keys to bring up rofi launcher. As far as I'm aware, you can't do that with Qtile, binding a single mod key to do an action.

**Comment by elParaguayo** [comment]
You can do it but have a read of #2778 to see some issues/limitations with it.

**Reply by vunhatchuong** [reply]
Yeah sadly the issues with ` Key([], "Super_L", lazy.spawn("rofi -show")),` make this workaround impossible to use.

**Reply by m-col** [reply]
Could you clarify what behaviour you're after - is it that you want the binding to trigger on key release, or for other multi-key bindings also using "Super_L" to work too (or both)?

**Reply by vunhatchuong** [reply]
I want a binding of mod4 alone to trigger on release, but when combines with other keys it trigger on press. 

For example in Windows, you can press Win key to open up start menu but you can also press Win+E to  open file explorer.

**Reply by hanschen** [reply]
On Linux/X11, you can use something like [xcape](https://github.com/alols/xcape) to get this functionality. Unfortunately it doesn't seem to be actively maintained anymore, but it seems to work well for me.
